docs/Quickstart.md

```diff
@@ -4,180 +4,392 @@
 
-To check the different configuration on the Catalog, please go to [Advanced Configuration](AdvancedConfiguration.md)
+### 
 
-```scala
-ecommerce.createOrReplaceTmpView("ecommerce_october")
+## Append
+
+Append data to a path using DataFrame API in “append” mode, or SQL Insert Into clause.
+
+### Python
+
+#### Append to a Table
+```python
+append = spark.createDataFrame([(4, "d"), (5, "e")], "id: int, age:string")
 
-spark.sql("CREATE OR REPLACE TABLE ecommerce_qbeast USING qbeast AS SELECT * FROM ecommerce_october")
+# Save
+append.write.\
+  mode("append").\
+  insertInto("qbeast_table")
+```
 
-//OR
+#### Append to a Path
+```python
+append = spark.createDataFrame([(4, "d"), (5, "e")], "id: int, age:string")
+
+# Save
+append.write.\
+  mode("append").\
+  option("columnsToIndex", "id,age").\
+  format("qbeast").\
+  save("/tmp/qbeast_table")
+```
 
-val ecommerceNovember = spark.read
-  .format("csv")
-  .option("header", "true")
-  .option("inferSchema", "true")
-  .load("./src/test/resources/ecommerce100K_2019_Nov.csv")
-  
-ecommerceNovember.createOrReplaceTmpView("ecommerce_november")
+### Scala
+
+#### Append to a Table
+```scala
+val append = Seq((4, "d"), (5, "e")).toDF("id", "age")
 
-spark.sql("INSERT INTO ecommerce_qbeast SELECT * FROM ecommerce_november")
+// Save
+append.write.
+  mode("append").
+  insertInto("qbeast_table")
 ```
-Sampling has also an operator called `TABLESAMPLE`, which can be expressed in both terms of rows or percentage. 
 
+#### Append to a Path
 ```scala
-spark.sql("SELECT avg(price) FROM ecommerce_qbeast TABLESAMPLE(10 PERCENT)").show()
+val append = Seq((4, "d"), (5, "e")).toDF("id", "age")
+
+// Save
+append.write.
+  mode("append").
+  option("columnsToIndex", "id,age").
+  format("qbeast").
+  save("/tmp/qbeast_table")
+```
+
+### SQL
+
+Use **`INSERT INTO`** to add records to the new table. It will update the index in a **dynamic** fashion when new data is inserted.
+
+```sql
+INSERT INTO table qbeast_table VALUES (4, "d"), (5, "e");
 ```
 
-## Python
+## Read
 
-Similar to Scala and SQL, you can also use PySpark to operate on Qbeast tables. 
-Here, you can find an end-to-end example following the same operations demonstrated earlier on this page.
+Read data from a Qbeast Table by specifying the paths or the table name.
+
+### Python
 
 ```python
-from pyspark.sql import SparkSession
+qbeast_df = spark.read.format("qbeast").load("/tmp/qbeast_table")
 
-# Initialize Spark session with Qbeast and Delta support, running locally
-spark = SparkSession.builder \
-    .appName("QbeastExample") \
-    .master("local[*]") \
-    .config("spark.jars.packages", "io.qbeast:qbeast-spark_2.12:0.7.0,io.delta:delta-spark_2.12:3.1.0") \
-    .getOrCreate()
+qbeast_df.sample(0.3).show()
```

Comment:
Shouldn't you explain what is sampling?

Reply:
Sure, I will come up with a few lines

Reply:
You can check the file again and let me know wdyt about the explanation 🙈